packages/authgear-react-native/src/ui_implementation.ts

```diff
@@ -1,19 +1,23 @@
 import { openAuthorizeURL, openAuthorizeURLWithWebView } from "./nativemodule";
 
 /**
+ * OpenAuthorizationURLOptions is options for {@link UIImplementation.openAuthorizationURL}.
+ *
  * @public
  */
 export interface OpenAuthorizationURLOptions {
-  /*
-   * The URL to open.
+  /**
+   * The target authorization URL.
    */
```

Comment:
Change the description to `The URL to be opened by the UIImplementation`.

This documentation is for UIImplementation implementor. So from their point of view, they want to know how to implement a UIImplementation. The documentation should guide them.

Reply:
ok